File: POOPlatzi/JAVA/Account.java
class Account{

    Integer id;
    String nombre;
    String documento;
    String password;
    String Email;

public Account ( String nombre, String documento,String Email){

    this.nombre=nombre;
    this.documento=documento;
    this.Email=Email;


}
public void  imprimirInfo (){

  System.out.println("id: "+ id + "\n" +  " nombre conductor: "+ nombre +  "\n" +  "TI"+ documento +
   "\n" + "password: "+ password + "\n" + "Correo:"+ Email);
}


} 

File: POOPlatzi/PHP/Account.php
<?php
class Account {
// aca basicamente declaro mis atributos, no es necesario el igual=integer etc, solamente con public $id;
//es suficiente, php por defecto identifica el tipo de dato 
   public $id;      
   public $nombre;  
   public $documento;  
   public $password;
   public $Email;  

public function __construct($nombre,$documento){

   $this->nombre= $nombre;
   $this->documento=$documento;

   }

} 
?>







